CHANGELOG

```diff
@@ -1,3 +1,22 @@
+Release 117 Biskupin (2019-10-02)
+---------------------------------
+Scala Stream Collector: Bump version to 0.17.0 (#4208)
+Scala Stream Collector: Add support fo TLS-encryption (#4085)
+Scala Stream Collector: Bump akka-http to 10.1.10 (#4186)
+Scala Stream Collector: Bump Scala version to 2.11.12 (#4206)
+Common: change Travis distribution to Trusty (#4214)
+Scala Stream Collector: Allow users to disable the default redirect endpoint (#4211)
```

Comment:
All changes should be grouped by component.

Within each component, feature changes should come first, followed by version bumps etc.

Similarly, commits around travis and other under-the-hood details should come last.